tests/test_basic.py

```diff
@@ -1,8 +1,37 @@
+import pytest
+
 import base62
 
+
+bytes_int_pairs = [
+    (b'\x00', 0),
+    (b'\x01', 1),
+    (b'\x01\x01', 0x0101),
+    (b'\xff\xff', 0xffff),
+    (b'\x01\x01\x01', 0x010101),
+    (b'\x01\x02\x03\x04\x05\x06\x07\x08', 0x0102030405060708),
+]
+
+
 def test_basic():
     assert base62.encode(0) == '0'
     assert base62.decode('0') == 0
 
     assert base62.encode(34441886726) == 'base62'
     assert base62.decode('base62') == 34441886726
+
```

Comment:
I think a roundtrip test would be good.

For example, if you pass the following test in 2 and 3, I think you will have addressed my issues.

```python
input_bytes = b'my bytes to encode'
base62_encoded = base62.encodebytes(input_bytes)
assert isinstance(base62_encoded, str)
output_bytes = base62.decodebytes(base62_encoded)
assert isinstance(output_bytes, bytes)
assert input_bytes == output_bytes
```